### DISCUSSION TITLE
How to use pop

### DISCUSSION THREAD

**Discussion by stargazer1984** [pimalaya/himalaya#503] [Q&A]
I am looking for a way to use himalaya  with the pop3 protocol for retrieving my mail but there seems to be no option to do so. Is this feature not available or am I overlooking something here? I want my mail to be downloaded and removed from the mail server daily, so pop would be my choice.

**Comment by soywod** [comment]
The POP protocol is not supported at the moment. It is planned, but it is not a priority since the demand is low. There is a promising low-level crate but it's not ready yet: https://github.com/duesee/pop3-codec. This could help us implementing the POP backend for Himalaya.
